### DISCUSSION TITLE
Midjourney 暫停免費試用

### DISCUSSION THREAD

**Discussion by sjh501** [1101-GS4719-C/Python-Community-GS4719-C#1239] [General]
在三月底熱門的 AI 圖像生成網站 Midjourney 在 Discord 宣布暫時停止免費試用。最初大家猜測是近期的假照片生成導致停止免費試用，但官方隨後解釋是因為大量用戶為取得免費使用而開設大量帳戶（每一 Discord 帳戶只能有 25 張的免費使用量），進而導致伺服器癱瘓。目前再次開放免費試用的時間尚未公布。

前幾年假新聞的議題十分火熱，而 AI 所生成的圖文或許又會造成新一波的爭論；另外，為取得免費使用造成網站癱瘓影響他人，背後的道德倫理也是值得深思。

**Comment by curtis1017** [comment]
我覺得作者的想法很好耶！我覺得兩者的可能性都很高，但我的看法比較偏向近期的假照片生成過多，我覺得假照片有時候會增加我們的娛樂，但也有很大的部分會造成不必要的誤會，所以我覺得這樣的政策雖然造成困擾，但使用者付費也是理所當然的。
